src/jvm_wrapper/registration/kt_property.h

```diff
@@ -54,14 +54,12 @@ JVM_INSTANCE_WRAPPER(KtProperty, "godot.core.KtProperty") {
 
     INIT_JNI_BINDINGS(
         INIT_JNI_METHOD(GET_KT_PROPERTY_INFO, "getKtPropertyInfo", "()Lgodot/core/KtPropertyInfo;")
-        INIT_JNI_METHOD(IS_REF, "isRef", "()Z")
```

Comment:
This method should also be removed from graal jni config.

Reply:
Done